packages/core/src/base-axis-chart.ts

```diff
@@ -368,17 +374,27 @@ export class BaseAxisChart extends BaseChart {
 		// If the <g class="y axis"> exists in the chart SVG, just update it
 		if (yAxisRef.nodes().length > 0) {
 			yAxisRef.transition(t)
-				// Casting to any because d3 does not offer appropriate typings for the .call() function
+				.attr("transform", function () {
+					if (rtl) {
+						return `translate(${chartSize.width}, 0)`;
+					} else { return `translate(0, 0)`; }
```

Comment:
Let's make this 3 lines instead of 1, more readable

Reply:
Noted